spec/controllers/concerns/string_redacter_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe 'StringRedacter' do
+  module Idv
+    class RedactTestController < ApplicationController
+      include StringRedacter
+    end
+  end
```

Comment:
so ruby blocks are weird, this is going to define `Idv::RedactRestController` in the global namespace.

When making classes in specs, I'd consider using the `Class.new` syntax so that the classes are anonymous and don't stay around:


```ruby
let(:controller) do
  Class.new(ApplicationController.new) do
    include StringRedacter
  end.new
end

# ...

expect(controller.redact_alphanumeric)...
```

(also this mixin/concern business... another reason I'd consider just a static method `StringRedacter.redact_alphanumeric`)